### DISCUSSION TITLE
Please consider Migration to Tauri

### DISCUSSION THREAD

**Discussion by LeSnake04** [Zettlr/Zettlr#3760] [Ideas]
Tauri is a great project that provides an alternative to Electron that heavily reduces ram usage, cpu usage, and greatly reduced file size (only a few Mb with all features enabled excluding  assets) by replacing chromium with rust and using the platforms native browser engine instead bundling a huge browser. And you can still write almost everything in javascript/typescript, so migration shouldnt be too hard and you dont need to learn too much rust (But since rust is way  faster and more stable, you might consider rewriting some error prone and performance heavy task in rust).

Recources:
https://tauri.app
https://github.com/mul14/electron-to-tauri

**Comment by nathanlesage** [comment]
Thanks for noticing me about the 1.0 release.

There will in the future be at some point a rewrite of the full codebase away from Electron as Electron also will be discontinued at some point or another. However, right now is not the time, and I cannot make any long-term plans in this direction. I do keep the cross-platform Electron-alternative sphere on my radar and monitor what's happening in there, but right now there is no way.

**Reply by gour** [reply]
> There will in the future be at some point a rewrite of the full codebase away from Electron as Electron also will be discontinued at some point or another. 

This is great news which might me seriously consider to simply start using Zettlr for my note-taking and/or writing knowing well that I can always migrate to some other native FLOSS editor. :wink:
